README.md

```diff
@@ -34,15 +34,53 @@ angular
     */ 
     var mySwaggerAPI = new AngularSwaggerific($window.swaggerJson);
 
-    MySwaggerAPI.{namespace}.{operationId}({"id": 1})
+    mySwaggerAPI.{namespace}.{operationId}(options)
       .then(function(data) { 
-        $log.log("Success! " + data);
+      	$log.log("Success! " + data);
       }, function(err) {
-        $log.log("Error! " + err);
-    });
+      	$log.log("Error! " + err);
+      });
+    
+    /**
+    * Pet Store Examples
+    */
+    
+    mySwaggerAPI.pet.getPetById({petId:1})
+      .then(function(data) {
+     	 $log.log("Success! Pet: " + data);
+      }, function(err) {
+     	 $log.log("Error! " + err);
+      });
+    
+    mySwaggerAPI.pet.findPetsByTags({params:{tags:"example"}})
+      .then(function(data) {
+     	 $log.log("Success! Pets: " + data);
+      }, function(err) {
+     	 $log.log("Error! " + err);
+      });
+      
+    
+    /**
+    * Cancel all current requests that are made through the swagger client
+    */
+    mySwaggerAPI.cancelRequests();
   }); 
 ```
 
+#### Possible Options
+
+In the options section, you put any of the variables that are passed in the URL, as they appear in the Swagger JSON. For instance, the Pet Store Swagger example has `/pet/{petId}` as a URL, so in the options, you would pass `petID` with the ID you are wanting.
+
++ params `[object]`
+  + Params are the variables that can be passed as a parameter.
+  + If it is a `GET` or `DELETE` request, they are appended to the URL
+  + If it is a `POST` or `PUT` request, they are sent as form data
++ ignoreDuplicateRequest `boolean`
```

Comment:
The naming here seems a bit ambiguous. `ignoreDuplicateRequests` does not mean what I initially thought before reading through the code. "Ignore" could refer to ignoring the result, or ignore cancelling.

I think something like `cancelDuplicateRequests` could work. And then in turn flipping the description for true and false. Also, I think we should default this to `true`.